wikicharts/wikicharts.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #382.                yoy_change_value = (yoy_highlight[str(y)].iat[-1] - yoy_highlight[str(y)].iat[0]) * 100  # added an opening parenthesis here
</pre><p>Hmm, this is interesting: it seems like you are giving the absolute year-over-year change in percentage points, rather than the relative year-over-year percent change (e.g. 5% last year, 10% now is an absolute increase of 5 percentage points, but a relative increase of 50%).</p><p> </p><p>Was this was the stakeholders asked for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>